### ISSUE TITLE
[BUG] Lair connection or setup fails with InvalidInput error

### ISSUE THREAD

**Issue by matthme** [holochain/launcher-tauri#144]
**Describe the bug**
According to a report, the Launcher freezes on a 2019 MacBook Pro with the following errors in the logs:
```
[2023-03-11T10:00:51.119178-07:00] ERROR - [LAIR] {"error":"InvalidInput","message":"path must be shorter than libc::sockaddr_un.sun_path"}
[2023-03-11T10:00:51.120572-07:00] INFO - [LAIR] Terminated(TerminatedPayload { code: Some(0), signal: None })
```

**Comment by matthme** [comment]
There is a limit to Unix domain socket paths. The path to lair's socket for the default profile is 86 characters without username already: `/Users/[username]/Application Support/holochain-launcher/profiles/default/0.2/socket`. Will need to find a solution for this.

**Comment by matthme** [comment]
Fixed with #145
